mne/decoding/csp.py

```diff
@@ -56,9 +57,9 @@ def __init__(self, n_components=4, reg=None, log=True):
         """Init of CSP."""
         self.n_components = n_components
         if reg == 'lws':
-            raise DeprecationWarning('`lws` has been deprecated for the `reg`'
-                                     ' argument. It will be removed in 0.11.'
-                                     ' Use `ledoit_wolf` instead.')
+            warnings.warn('`lws` has been deprecated for the `reg`'
+                          ' argument. It will be removed in 0.11.'
+                          ' Use `ledoit_wolf` instead.', DeprecationWarning)
```

Comment:
@trachelr should use `warnings.warn` instead of `raise DeprecationWarning`, the `raise` will actually halt execution